src/RpcClient/Nep5API.cs

```diff
@@ -117,5 +118,32 @@ public Transaction CreateTransferTx(UInt160 scriptHash, KeyPair fromKey, UInt160
 
             return tx;
         }
+
+        /// <summary>
+        /// Create NEP5 token transfer transaction from multi-sig account
+        /// </summary>
+        /// <param name="scriptHash">contract script hash</param>
+        /// <param name="m">multi-sig min signature count</param>
+        /// <param name="pubKeys">multi-sig pubKeys</param>
+        /// <param name="fromKeys">sign keys</param>
+        /// <param name="to">to account</param>
+        /// <param name="amount">transfer amount</param>
+        /// <returns></returns>
+        public Transaction CreateTransferTx(UInt160 scriptHash, int m, ECPoint[] pubKeys, KeyPair[] fromKeys, UInt160 to, BigInteger amount)
+        {
+            if (m > fromKeys.Length) throw new System.Exception($"Need at least {m} KeyPairs for signing!");
```

Comment:
|| m<0 ?

Reply:
Maybe this is also an ArgumentExcception.

![image](https://user-images.githubusercontent.com/4393234/78646866-4a286e00-78ec-11ea-9979-d1d13fb86110.png)